install/Knative-with-Minikube.md

```diff
@@ -9,8 +9,9 @@ You can find [guides for other platforms here](README.md).
 
 ## Before you begin
 
-Knative requires a Kubernetes cluster v1.11 or newer. If you don't have one, you
-can create one using [Minikube](https://github.com/kubernetes/minikube).
+While Knative requires a Kubernetes cluster of v1.11 or newer in genereal, it requires a Kubernetes
```

Comment:
```suggestion
While Knative requires a Kubernetes cluster of v1.11 or newer in general, Minikube requires a Kubernetes
```

Reply:
@samodell by `it` I refer to `knative` not `Minikube`